afk.lic

```diff
@@ -46,10 +46,11 @@ loop do
   end
 
   if line =~ /For a current status of how many items you have, use LOOK PORTAL./
-    echo "Looks like you're in the junk yard.  We'll log you out in 60 seconds.  You can ';kill afk' to stay in the game, just remember to restart it once you get out of the junk yard"
-    pause 60
-
-    exit_game('Exiting from the Junk Yard') if XMLData.room_title == '[[A Junk Yard]]'
+    echo "Looks like you're in the junk yard.  We'll log you out in 60 seconds unless you exit the junk yard.  You can also ';kill afk' to stay in the game, just remember to restart it once you get out of the junk yard."
+    junk_timer = Time.now
+    while XMLData.room_title == '[[A Junk Yard]]'
```

Comment:
❓ Does a pause-less `while` cause any CPU issues? Would a `pause 1` be good here?